waspc/data/Generator/templates/server/src/queries/types.ts

```diff
@@ -0,0 +1,33 @@
+{{={= =}=}}
```

Comment:
You can throw in a comment in both of these files thta they are very similar to each other. I did this in some other places. It means that we know about duplication, but we are ok with it. It also point whoever might be refactoring file to also look at the other one.